addons/account/views/partner_view.xml

```diff
@@ -75,7 +75,7 @@
                                         domain="[
                                             ('type_tax_use', '!=', 'none'),
                                             ('country_id', '=', parent.country_id if parent.foreign_vat else parent.company_country_id),
-                                            '|', ('company_id', '=', False), ('company_id', '=', parent.company_id)]"
+                                            '|', ('company_id', '=', False), ('company_id', 'parent_of', parent.company_id)]"
```

Comment:
I'm just realizing right now.

Shouldn't it be `parent_of company_id` and not `parent_of parent.company_id`?

Reply:
Nope.
`parent.company_id` seems correct as it is the company of the fiscal position.